src/manageiq_client/api.py

```diff
@@ -622,6 +622,15 @@ def _process_result(self, result):
             if "id" in result:
                 d["href"] = "{}/{}".format(collection._href, result["id"])
             return Entity(collection, d)
+        elif "task_href" in result or "task_id" in result:
+            collection = getattr(self.api.collections, "tasks")
+            # reuse task_href and task_id, no other data is relevant
+            d = {}
+            if result.get("task_href"):
+                d["href"] = result["task_href"]
+            if result.get("task_id"):
+                d["id"] = result["task_id"]
```

Comment:
@mkoura would this be nicer? (I know it's not the same, would it be problematic?)

```python
d = {
    'id': result.get('task_id')
    'href': result.get('task_href')
}
```

Do you think in the future we'll want to return more info? (without having to update this code)